tools/interactive-evdev.c

```diff
@@ -499,6 +509,8 @@ main(int argc, char *argv[])
         {"consumed-mode",        required_argument,      0, OPT_CONSUMED_MODE},
         {"enable-compose",       no_argument,            0, OPT_COMPOSE},
         {"print-keycode",        no_argument,            0, OPT_PRINT_KEYCODE},
+        {"hide-layout",          no_argument,            0, OPT_HIDE_LAYOUT},
```

Comment:
I feel like adding these options is not ideal.

Is the problem that it becomes too long that wraps the terminal or something else?

Reply:
Yes, this is the problem.

Reply:
The "hide" approach is not very scalable. I think we should either go all-in and allow configuring the exact set of fields generically, or go the other way and only add a `--short` option which makes the output shorted in an unspecified manner. Personally I'd go with `--short`.

Reply:
Goo idea. Done.